internal/database/postgres/pg.go

```diff
@@ -167,7 +173,11 @@ func (c Config) String(useAdmin bool, appName string) string {
 	if !useAdmin {
 		fields = append(fields, "dbname="+c.Database)
 	} else {
-		fields = append(fields, "dbname=postgres")
+		defaultDB := c.Admin.ExistingDatabase
+		if defaultDB == "" {
+			defaultDB = "postgres"
```

Comment:
I think this would change the current behavior i would just leave it empty if there is no database name specified

Reply:
Yeah, that's better

Reply:
I agree if you mean the value *defaultdb* for cockroach in the other file. postgres was the constant before, so having that as default doesn't change the behavior.

No spec defaults to *defaultdb*, but leaving it empty there altogether is better👍